src/Parser.zig

```diff
@@ -5583,7 +5586,8 @@ fn offsetofMemberDesignator(p: *Parser, base_ty: Type) Error!Result {
             }
             try p.validateFieldAccess(lhs.ty, lhs.ty, field_name_tok, field_name);
             const record_ty = lhs.ty.canonicalize(.standard);
-            lhs = try p.fieldAccessExtra(lhs.node, record_ty, field_name, false);
+            lhs = try p.fieldAccessExtra(lhs.node, record_ty, field_name, false, &offset_num);
+            bit_offset = Value.int(offset_num);
```

Comment:
This should check if `bit_offset != .unavailable` and then add to it rather than setting it.

Reply:
added check, and addation.